tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -183,7 +186,7 @@ describe(__filename, () => {
         undefined,
         loadAddonsByAuthors({
           addons: [fakeAddon],
-          authorUsernames: [fakeAddon.authors[0].username],
+          authorIds: [fakeAddon.authors[0].id],
```

Comment:
Just looking at this test in isolation, there's no guarantee that `fakeAddon.authors[0].id` isn't the same as `randomAuthorId1`. Maybe it would be better to have 2 new consts in this test for the ids used here and in the second call to `fetchAddonsByAuthors` so that it's obvious to the test that they are different values. I don't think it actually matters that this is `fakeAddon.authors[0].id` here - it looks like it could be any number.

Reply:
Having said all that, it actually looks like the `authorIds` are irrelevant to this test. It would behave the same way if the authors are the same or different, so maybe it should be rewritten to be more simpler, as it suggests that they are expected to be different in the test.

Reply:
I set the same `authorIds` for both actions, but I don't really know what this test case is doing tbh. I just `s/username/id` here. That's probably not smart, but I prefer that over introducing a bug/weakness.